pgppubkey_to_jwk.py

```diff
@@ -1,17 +1,9 @@
-from authlib.common.encoding import int_to_base64
 import json
-import pgpy
+from josepy import JWK
 
+with open('public_release.pem', 'rb') as f:
```

Comment:
using an armored PGP public key as input here doesn't seem to work (output of the script ends up being a JSON object with  a `"kty": "RSA"` entry, but without `e`, `n` fields), is there something we need to do in josepy to handle the armored PGP public key case? 

if no that's fine, we can just keep this script as separate (its purpose is to convert pgp pubkey -> jwk object) and add back those required dependencies. 

basically we want to continue to handle two cases:
* one that takes PGP key as input, produces valid JWK (this script)
* one that takes PEM, produces valid JWK, and is used in `make test-key` (in `Makefile`)